bofire/strategies/doe_strategy.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+from pydantic.types import PositiveInt
+
+import bofire.data_models.strategies.api as data_models
+from bofire.strategies.doe.design import find_local_max_ipopt
+from bofire.strategies.strategy import Strategy
+
+
+class DoEStrategy(Strategy):
+    """Strategy for design of experiments. This strategy is used to generate a set of
+    experiments for a given domain.
+    The experiments are generated via minimization of the D-optimality criterion.
+
+    """
+
+    def __init__(
+        self,
+        data_model: data_models.DoEStrategy,
+        **kwargs,
+    ):
+        super().__init__(data_model=data_model, **kwargs)
+        self.formula = data_model.formula
+        self.domain = data_model.domain
+
+    def _ask(self, candidate_count: PositiveInt) -> pd.DataFrame:
+        _fixed_experiments = self.candidates
+        _domain = self.domain
+        design = find_local_max_ipopt(
+            _domain,
+            self.formula,
+            n_experiments=candidate_count,
+            fixed_experiments=_fixed_experiments,
+        )
+        return design  # type: ignore
+
+    def has_sufficient_experiments(
+        self,
+    ) -> bool:
+        """Abstract method to check if sufficient experiments are available.
+
+        Returns:
+            bool: True if number of passed experiments is sufficient, False otherwise
+        """
+        return True
+
+    def set_candidates(self, candidates: pd.DataFrame):
```

Comment:
why, this? Do we do not have this already?

Reply:
https://github.com/experimental-design/bofire/blob/5d3c2e7c7887daf773621784671826c5602b00f0/bofire/strategies/strategy.py#L190

Reply:
the existing one does not have the only_inputs=True flag

Reply:
This is a bug in the original one, can you remove yours and add in the original one `only_inputs=True`?